### DISCUSSION TITLE
Support for Web

### DISCUSSION THREAD

**Discussion by nikhilnowfloats** [vitusortner/floor#762] [Q&A]
We have been using floor for mobile application. At the same time we are developing the desktop and web application for the same project. Can we expect support for web platform as well in the near future?

**Comment by dkaera** [comment]
> We have been using floor for mobile application. At the same time we are developing the desktop and web application for the same project. Can we expect support for web platform as well in the near future?

there is PR #730 , and we need to add a simple test to ensure that the test coverage action passes. 
As option, you can use a forked branch.

**Reply by michasng** [reply]
Given that the PR has been merged, does this mean web is now officially supported?
(if so, then the docs are outdated: https://pinchbv.github.io/floor/platform-support/)
